prophy/composite.py

```diff
@@ -260,6 +260,44 @@ def field_to_string(name, type_, value):
     else:
```

Comment:
"fields" and "values" do very specific thing, probably one party of prophy users will make use of it:
1) get_descriptor() function in struct/union generated classes
   this part is generic, will form a stable interface for users
   who want reflection (iteration over fields, figuring out their type, etc.)
2) values() and fields() functions which are defined at user side (not in prophy),
   which may work in generic way with all structs and unions